README.md

```diff
@@ -0,0 +1,175 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="" alt="logo" width="400"  height="auto" />
+  <br/>
+  <img src="https://user-images.githubusercontent.com/65084923/221706474-1ec353fc-9a3f-4a0f-a772-da8b2cafb1f0.png" alt="logo" width="300"  height="auto" />
+  <br/>
+  <img src="https://user-images.githubusercontent.com/65084923/220200035-01fa5e08-af6a-495b-b597-55157a2b7d95.gif" alt="logo" />
+  <br/>
+
+  <h3><b>React To Do List</b></h3>
+  <h3><b>React To Do List - Microverse Project Module 3</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [🧑🏻‍💻 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Available Scripts](#available-scripts)
+  - [Clone](#clone)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 React To Do List - Microverse Project Module 3<a name="about-project"></a>
+
+> Math Magicians React App developed with Nodejs, Html, Css, Js, Webpack, React
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">Html</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">Css</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/JavaScript">Vanilla Javascript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+    <li><a href="https://react.org/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Task**
+- **Edit Task**
+- **Delete Task**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧑🏻‍💻 Live Demo <a name="live-demo"></a>
+
+coming soon...
+<br/>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template for your own portafolio site, follow these steps.
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/chelobotix/Math-Magicians-Chelobotix.git
+```
+
+## 🤖 Available Scripts <a name="available-scripts"></a>
+
+This project use a Vite REact Template:
+
+### `npm create vite@latest`
+
+In the project directory, you can run:
+
+### `npm i`
+
+Runs the app in the development mode.\
+Open [http://localhost:5173](http://localhost:5173) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```

Comment:
Running this command generates an error. This is because in your package.json file, there is noo script for it. Please correct this or remove it totally from the readme